Pock/Widgets/NowPlaying/NowPlayingItemView.swift

```diff
@@ -36,7 +36,7 @@ class NowPlayingItemView: PKDetailView {
         switch (appBundleIdentifier) {
         case "com.apple.WebKit.WebContent":
             appBundleIdentifier = "com.apple.Safari"
-        case "com.spotify.client", "com.apple.iTunes", "com.apple.Safari":
```

Comment:
added [Netease music](https://music.163.com/) support , which is the most common music app in China